_scripts/app1-6.R

```diff
@@ -0,0 +1,28 @@
+{
+  
+  rm(list=ls());
+  options(show.error.locations = TRUE);
+  
+ # Prompt user to input location 
+  
+  tempLoc <- readline("Where were the measurements recorded?: ");
+  
+ # Prompt user to input recorded measurements
+  
+  temp1 <- as.numeric(readline("Enter the first measurement: "));
+  temp2 <- as.numeric(readline("Enter the second measurement: "));
+  temp3 <- as.numeric(readline("Enter the third measurement: "));
+  temp4 <- as.numeric(readline("Enter the fourth measurement: "));
+  temp5 <- as.numeric(readline("Enter the fifth measurement: "));
+  
+  # Calculate mean temperature
+  
+  mnTemp <- sum(temp1, temp2, temp3, temp4, temp5)/5; # n = 5
+  
+  # Summarize in console
+  
+  cat("The temperature measurements are from: ", tempLoc,
+      "\nThe recorded temperatures were: ", temp1, temp2, temp3, temp4, temp5,
+      "\nThe average temperature was: ", mnTemp);
```

Comment:
For formatting purposes in the Console, it often helps to add "\n" to the end of cat():
  "\nThe average temperature was: ", mnTemp, "\n");

To see why, add to line 27:
print(temp1);

Reply:
@belinskyc

Would this apply to the beginning of the cat('') statement as well? 
I.e. cat("\n....\n")

Reply:
"\n" is the programming equivalent of hitting enter so it all depends on what was outputted to the Console right before cat().  Putting in the extra \n is safe.  You can always go back and delete the \n  if there is too much space.